app/controllers/guides_controller.rb

```diff
@@ -25,17 +25,14 @@ def new
   def create
     @guide = Guide.new(guide_params)
 
-    ActiveRecord::Base.transaction do
-      if @guide.save
-        GuidePublisher.new(guide: @guide).put_draft
-        redirect_to edit_guide_path(@guide), notice: "Guide has been created"
-      else
-        render action: :new
-      end
+    publication = Publisher.new(content_model: @guide).
```

Comment:
Is that style guide stuff or should this dot be on a new line. Sorry that's incredibly pedantic